temporalcli/commandsmd/code.go

```diff
@@ -319,12 +319,26 @@ func (c *CommandOption) writeFlagBuilding(selfVar, flagVar string, w *codeWriter
 	default:
 		return fmt.Errorf("unrecognized data type %v", c.DataType)
 	}
+
+	// If there are enums, append to desc
+	desc := c.Desc
+	if len(c.EnumValues) > 0 {
+		desc += " Accepted values: "
+		for i, enumVal := range c.EnumValues {
+			if i > 0 {
+				desc += ", "
+			}
+			desc += enumVal
+		}
+		desc += "."
+	}
```

Comment:
```suggestion
	if len(c.EnumValues) > 0 {
		c.Desc += fmt.Sprintf(" Accepted values: %s.", strings.Join(c.EnumValues, ", "))
	}
```

Reply:
Yeah, I did this originally because I had "and" in there and it didn't look right, will change.